Source/Serilog.Enrichers.Span/SpanOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace Serilog.Enrichers.Span
+{
+    /// <summary>
+    /// Options.
+    /// </summary>
+    public class SpanOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to include tags in log or not.
+        /// </summary>
+        public bool IncludeTags { get; set; }
```

Comment:
We should default this to `false` and mention that in the comment.

Reply:
Setting this to `false` would trigger a warning, since the default value would be `false`.

Reply:
Ah of course, sorry.